src/entries/popup/components/TransactionFee/TransactionFee.tsx

```diff
@@ -123,18 +123,21 @@ function Fee({
   useKeyboardShortcut({
     handler: (e: KeyboardEvent) => {
       if (e.key === shortcuts.global.OPEN_CUSTOM_GAS_MENU.key) {
-        trackShortcut({
-          key: shortcuts.global.OPEN_CUSTOM_GAS_MENU.display,
-          type: 'customGasMenu.open',
-        });
-        // hackery preventing GweiInputMask from firing an onChange event when opening the menu with KB
```

Comment:
Should keep this comment probably to document the nav hack

Reply:
+1

Reply:
oops